src/Microsoft.AspNetCore.Mvc.Core/Internal/IResponseCacheFilter.cs

```diff
@@ -6,9 +6,9 @@
 namespace Microsoft.AspNetCore.Mvc.Internal
 {
     /// <summary>
-    /// An <see cref="IActionFilter"/> which sets the appropriate headers related to Response caching.
+    /// A filter which sets the appropriate headers related to Response caching.
     /// </summary>
-    public interface IResponseCacheFilter : IActionFilter
```

Comment:
Use a different marker type for pages? I thought we were kinda ok changing this since it's in `.Internal`